llmeval/setup_node.sh

```diff
@@ -8,6 +8,7 @@ fi
 
 export VENV_DIR=$1
 export HF_HOME=$2
+export HF_TOKEN=$3
```

Comment:
We dont need to pass it if we use source, it can be defined in the environment where `setup_node.sh` is called.
```suggestion
```